src/NzbDrone.Core.Test/MediaFiles/EpisodeImport/Specifications/EpisodeTitleSpecificationFixture.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using FizzWare.NBuilder;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Core.MediaFiles.EpisodeImport.Specifications;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Test.Framework;
+using NzbDrone.Core.Tv;
+using NzbDrone.Test.Common;
+
+namespace NzbDrone.Core.Test.MediaFiles.EpisodeImport.Specifications
+{
+    [TestFixture]
+    public class EpisodeTitleSpecificationFixture : CoreTest<EpisodeTitleSpecification>
+    {
+        private Series _series;
+        private LocalEpisode _localEpisode;
+
+        [SetUp]
+        public void Setup()
+        {
+            _series = Builder<Series>.CreateNew()
+                                     .With(s => s.SeriesType = SeriesTypes.Standard)
+                                     .With(s => s.Path = @"C:\Test\TV\30 Rock".AsOsAgnostic())
+                                     .Build();
+
+            var episodes = Builder<Episode>.CreateListOfSize(1)
+                                           .All()
+                                           .With(e => e.SeasonNumber = 1)
+                                           .With(e => e.AirDateUtc = DateTime.UtcNow)
+                                           .Build()
+                                           .ToList();
+
+            _localEpisode = new LocalEpisode
+                                {
+                                    Path = @"C:\Test\Unsorted\30 Rock\30.rock.s01e01.avi".AsOsAgnostic(),
+                                    Episodes = episodes,
+                                    Series = _series
+                                };
+        }
+
+        [Test]
+        public void should_reject_when_title_is_null()
+        {
+            _localEpisode.Episodes.First().Title = null;
+
+            Subject.IsSatisfiedBy(_localEpisode, null).Accepted.Should().BeFalse();
+        }
+
+        [Test]
+        public void should_reject_when_title_is_TBA()
+        {
+            _localEpisode.Episodes.First().Title = "TBA";
+
+            Subject.IsSatisfiedBy(_localEpisode, null).Accepted.Should().BeFalse();
+        }
+
+        [Test]
+        public void should_reject_when_title_is_a_placeholder()
+        {
+            _localEpisode.Episodes.First().Title = $"Episode {_localEpisode.Episodes.First().EpisodeNumber}";
```

Comment:
No, should accept, coz these are valid titles and there are countless of series that do this.